build-parent/pom.xml

```diff
@@ -541,6 +545,38 @@
                         </buildArgs>
                     </configuration>
                 </plugin>
+                <plugin>
+                    <groupId>io.fabric8</groupId>
+                    <artifactId>docker-maven-plugin</artifactId>
+                    <version>${version.fabric8.docker.maven.plugin}</version>
+                    <extensions>true</extensions> 
+                    <executions>
+                        <execution>
+                            <id>default-build</id>
+                            <configuration>
+                                <images>
+                                    <image>
+                                        <name>${docker.baseline.repo.id}/${dockerImageName}:${dockerImageVersion}</name>
```

Comment:
I: By including the `${docker.baseline.repo.id}` in our image name, `fabric8` knows to push the image to that docker repo. See [fabric8 docs](https://dmp.fabric8.io/#registry) for more details.